src/components/Button.tsx

```diff
@@ -5,10 +5,14 @@ export class Button extends React.Component<{
   label?: string;
   title?: string;
   isDisabled?: boolean;
-  onClick?: Function
+  onClick?: Function;
+  customClassName?: string;
 }, {}> {
   render() {
     let className = "button";
+    if (this.props.customClassName) {
+      className += this.props.customClassName;
```

Comment:
As @mbebenita said, there should be a space in between classes

Reply:
Add that in linting. It doesn't show me any issue.